cartographer_ros/cartographer_ros/assets_writer_main.cc

```diff
@@ -55,7 +55,8 @@ int main(int argc, char** argv) {
       << "-pose_graph_filename is missing.";
 
   ::cartographer_ros::AssetsWriter asset_writer(
-      FLAGS_pose_graph_filename, absl::StrSplit(FLAGS_bag_filenames, ','),
+      FLAGS_pose_graph_filename,
+      absl::StrSplit(FLAGS_bag_filenames, ',', absl::SkipWhitespace()),
```

Comment:
I think it is much better to use `absl::SkipEmpty()` here. `" "` is a valid filename for which this tool now no longer works correctly, i.e. I would consider this a bug.

Reply:
Ok. Let's change it before Wally wakes up from the dead.

Reply:
Done.